packages/block-library/src/search/editor.scss

```diff
@@ -1,11 +1,39 @@
 .wp-block[data-align="center"] {
 	.wp-block-search {
+		float: unset;
+		display: flex;
+		align-items: center;
+		flex-direction: column;
+		margin-left: 0;
 		.wp-block-search__inside-wrapper {
 			margin: auto;
 		}
 	}
 }
 
+.wp-block[data-align="right"] {
+	.wp-block-search {
+		float: unset;
+		display: flex;
+		align-items: flex-end;
+		flex-direction: column;
+		margin-left: 0;
+	}
+	.wp-block-search__label {
+		min-width: 220px !important;
```

Comment:
This is ugly, but the RichText component adds a minWidth of 1px as an inline style with no option to specify a custom width, if we go with this approach we should probably add a minWidth prop to this component, the native version has this.

Reply:
another option would be to add an extra div around the label and input box and assign the width to this instead